snapcraft/tests/test_cmds.py

```diff
@@ -31,12 +30,6 @@
 )
 
 
-class _IO(io.StringIO):
-
-    def fileno(self):
-        return 1
```

Comment:
It was important when trying to mock stdout for subprocess, that code was killed, so this is not used anymore.